ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -156,13 +156,19 @@ export const isBypassed = (
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
+// Since there is no expiration date for ERA COMMONS or GOOGLE 2-Step Verification,
```

Comment:
This comment no longer applies since we're not hard-coding these

Reply:
Yes, however i would prefer to tweek it a bit rather than removing it, just to give a context whey we are introducing canExpire and which modules it could be

Reply:
yes, that is a nice improvement